src/js/experiments/index.js

```diff
@@ -3,11 +3,13 @@ import highlightjsBadge from './highlightJSBadge';
 import relatedResources from './relatedResources';
 import surfaceFullExampleConfig from './surfaceFullExampleConfig';
 import videoTutorials from './videoTutorials';
+import snippetFeedback from './snippetFeedback';
 
 export default {
   forceAll,
   highlightjsBadge,
   relatedResources,
   surfaceFullExampleConfig,
   videoTutorials,
+  snippetFeedback,
```

Comment:
I think we may have to import this as `SnippetFeedback` ?  Was getting a console warning on it

Reply:
Can you post the console warning you are getting?